examples/hello_services/android/app/build.gradle

```diff
@@ -7,6 +7,7 @@ android {
 
     lintOptions {
         disable 'InvalidPackage'
+        abortOnError false
```

Comment:
And... I don't see the lint / javac error that was failing the build anymore. I'll remove this line.